src/reason-parser/reason_lexer.mll

```diff
@@ -538,9 +538,9 @@ rule token = parse
     set_lexeme_length lexbuf 1;
     LBRACE
   }
-  | "</" uppercase_or_lowercase (identchar | '.') * ">" {
+  | "</" blank* uppercase_or_lowercase (identchar | '.') * blank* ">" {
     let buf = Lexing.lexeme lexbuf in
-    LESSSLASHIDENTGREATER (String.sub buf 2 (String.length buf - 2 - 1))
+    LESSSLASHIDENTGREATER (String.trim (String.sub buf 2 (String.length buf - 2 - 1)))
```

Comment:
Unrelated question: how many tries did it take you to get this right? This string api is the worst

Reply:
hahaha. I typed `String.` and searched for `trim`. Merlin helped me there, so 1 try!

Reply:
But the calculation, etc? String.sub throws so I'm always uneasy with off-by-ones

Reply:
The calculation was there already. I'm just trimming the result. What the calculation does is remove `"</"` and `">"` from what we lexed. I'm trimming that result.